ci/build.yml

```diff
@@ -164,9 +164,9 @@ jobs:
     - bash: |
         set -euo pipefail
         eval "$(./dev-env/bin/dade-assist)"
-        bazel build --config scala_2_13 -- //libs-scala/... //daml-lf/data/... //daml-lf/language/... //daml-lf/transaction/... //daml-lf/validation/... //daml-lf/parser/... //daml-lf/interpreter/... -//daml-lf/interpreter/perf/...
+        bazel build --config scala_2_13 -- //libs-scala/... //daml-lf/data/... //daml-lf/language/... //daml-lf/transaction/... //daml-lf/validation/... //daml-lf/parser/... //daml-lf/interpreter/...
         # gatling-utils tests fail with a ClassNotFoundException for scala.collection.SeqLike
-        bazel test --config scala_2_13 -- //libs-scala/... //daml-lf/data/... //daml-lf/language/... //daml-lf/transaction/... //daml-lf/validation/... //daml-lf/parser/... //daml-lf/interpreter/... -//libs-scala/gatling-utils/... -//daml-lf/interpreter/perf/...
+        bazel test --config scala_2_13 -- //libs-scala/... //daml-lf/data/... //daml-lf/language/... //daml-lf/transaction/... //daml-lf/validation/... //daml-lf/parser/... //daml-lf/interpreter/... -//libs-scala/gatling-utils/...
```

Comment:
Maybe time for some multiline bash with `\`? Not sure on the yaml rules here.

Reply:
I was kind of hoping to go to //daml-lf/... as the next step to just reduce line length but if it does get longer instead, I’ll definitely add some newlines :slightly_smiling_face: